.github/workflows/node.js.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: "16"
+          node-version: "18"
```

Comment:
probably makes sense to upgrade to Node 18 if that's what we are currently building with, just wanted to highlight that we are doing this here

Reply:
The goal is to keep things compatible with the latest LTS and I think that switches over from 16 -> 18 in the next week or two. Whether that's sustainable long-term we'll have to see (there's some work in the pipeline to merge all of the MFs into a single monorepo which should make these tooling changes more straight-forward).